test/unit/src/common/components/user-avatar/t_user-avatar.js

```diff
@@ -1,22 +1,41 @@
 import React from 'react';
 import expect from 'expect';
 import { shallow } from 'enzyme';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
 
-import { UserAvatar } from '../../../../../../src/common/components/user-avatar';
+import UserAvatar from '../../../../../../src/common/components/user-avatar';
 import { HeadingOne, HeadingThree } from '../../../../../../src/common/components/vanilla/heading';
 
+const middlewares = [ thunk ];
+const mockStore = configureMockStore(middlewares);
+
 describe('<UserAvatar />', function() {
   let element;
+  let store;
+  let auth;
+  let user;
+
+  beforeEach(() => {
+    store = mockStore({
+      auth,
+      user
+    });
+
+    // shallow render container component with the store
+    element = shallow(<UserAvatar store={store} />)
```

Comment:
It's not very straight forward, because shallow render doesn't render child component, we need to find wrapped component by name and shallow render it too. But at least we are testing component connected to the store and we also see if properties are mapped as they should.